indexer/editable_map_object.cpp

```diff
@@ -744,6 +766,124 @@ bool EditableMapObject::ValidateWebsite(string const & site)
   return true;
 }
 
+//static
+bool EditableMapObject::ValidateFacebookPage(string const & facebookPage)
+{
+  if (facebookPage.empty())
+    return true;
+  //check that facebookPage contains valid username. See rules: https://www.facebook.com/help/105399436216001
+  if (strings::EndsWith(facebookPage, ".com") || strings::EndsWith(facebookPage, ".net"))
```

Comment:
https://www.facebook.com/alexander.net

Reply:
Hm, Facebook documentation tells `alexander.net` is invalid. Changed validation code to allow such values.